### PR TITLE
API wrapper generator changes for VPC

### PR DESCRIPTION
**Pull request by ksamoray**
Add VPC to context and handle it accordingly in tests and wrappers.

### CODE REVIEW

File: nsxt/utils.go [hunk 1]
```diff
@@ -680,7 +680,25 @@ func handlePagination(lister func(*paginationInfo) error) (int64, error) {
...
+			Required:     true,
+			ForceNew:     true,
+			ValidateFunc: validation.StringIsNotWhiteSpace,
+		},
+	}
+	if isVPC {
+		elemSchema["vpc_id"] = &schema.Schema{
+			Type:         schema.TypeString,
+			Description:  "Id of the VPC which the resource belongs to.",
+			Optional:     true,
```
Comment (annakhm) [inline nsxt/utils.go:697]: For VPC-specific resources, `vpc_id` should be required, is that correct? Should this helper be able to return schema with required `vpc_id`?

File: api/api_list.yaml [hunk 2]
```diff
@@ -986,7 +989,6 @@
     - New
     - Get
     - Delete
-    - List
```
[the remark this answers is not in the document]
Comment (ksamoray) [reply api/api_list.yaml:989]: This was specified twice within that object (line 992) - it's a bug. That failed the generator (probably added at some point by mistake without running the generator).

[the remark this answers is not in the document]
Comment (salv-orlando) [reply api/api_list.yaml:989]: Got it I did not see the duplicatin

**Review by salv-orlando — APPROVED** [review]
LGTM, let's start getting this in.

**Review by annakhm — APPROVED** [review]

**Pull request merged**